packages/eslint-plugin-gestalt/src/prefer-box-as-tag.test.js

```diff
@@ -0,0 +1,55 @@
+// @flow strict
+import { RuleTester } from 'eslint';
+import { readFileSync } from 'fs';
+import path from 'path';
+import rule, { SUPPORTED_HTML_TAGS } from './prefer-box-as-tag.js';
+
+const ruleTester = new RuleTester();
```

Comment:
We can pass in the parserOptions to `RuleTester` directly so we don't have to repeat them separately for the invalid or valid test cases:

```
const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2015 } });
```

See https://eslint.org/docs/developer-guide/nodejs-api#ruletester

Reply:
DONE!